supabase/migrations/20260808191240_cli_usage.sql

```
@@ -0,0 +1,28 @@
+  created_at timestamptz NOT NULL DEFAULT now(),
+  cli_version text NOT NULL,
+  command text NOT NULL DEFAULT '',
+  node_version text NOT NULL DEFAULT '',
+  os_platform text NOT NULL DEFAULT '',
+  apikey_id uuid NULL,
+  org_id uuid NULL,
+  source text NOT NULL DEFAULT 'config',
+  api_version text NOT NULL DEFAULT ''
+);
+CREATE INDEX cli_usage_created_at_idx ON public.cli_usage (created_at DESC);
+CREATE INDEX cli_usage_cli_version_idx ON public.cli_usage (cli_version);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)